src/core/parser_internal/docgen_worker.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "../symbol.hpp"
```

Comment:
relative path from current file dir is discouraged. It's best to always write relative the include path that you will give. In this case, everything should be w.r.t. `src/` like `core/symbol.hpp`.

Reply:
I actually agree with this, but then I think we should stick with it—for example, in `lexer_routines.hpp`, I notice we have `io/file_reader.hpp`, but then we also have `token.hpp`, `symbol.hpp`, etc., which are relative paths. We should change all includes to be relative to `src` to be less confusing—I'll bring this up in the other pr.

Reply:
oh wow I didn't even realize that was a relative path. But sure we can change everything to be relative to `src`.